pkg/autoscaler/scaling/multiscaler.go

```diff
@@ -70,6 +70,10 @@ type DeciderSpec struct {
 	StableWindow time.Duration
 	// The name of the k8s service for pod information.
 	ServiceName string
+	// InitialScale is the calculated initial scale of the revision, taking both
+	// revision initial scale and cluster initial scale into account. Revision initial
+	// scale overrides cluster initial scale.
+	InitialScale int32
```

Comment:
do we have to worry about existing deciders since they'll default to 0 rather than 1 here I think?

Reply:
MakeDecider is called on every kpa reconciling call, so when the kpa reconciling call is triggered, we will update the existing deciders with the right InitialScale value when this is deployed